File: app/src/main/java/com/mytaxiapp/ui/listvehicles/VehicleView.java
package com.mytaxiapp.ui.listvehicles;

import com.mytaxiapp.data.model.Placemark;
import com.mytaxiapp.ui.base.MvpView;

import java.util.ArrayList;

/**
 * @author Tosin Onikute.
 */

public interface VehicleView extends MvpView {

    void setAdapter(ArrayList<Placemark> vehicleItemList);

    void showLoading();

    void hideLoading();
}

File: app/src/main/java/com/mytaxiapp/ui/listvehicles/VehicleView.kt
package com.mytaxiapp.ui.listvehicles

import com.mytaxiapp.data.model.Placemark
import com.mytaxiapp.ui.base.MvpView

import java.util.ArrayList

/**
 * @author Tosin Onikute.
 */

interface VehicleView : MvpView {

    fun setAdapter(vehicleItemList: ArrayList<Placemark>)

    override fun showLoading()

    override fun hideLoading()
}
